<html>
  <head>
  <!-- put your analytics (e.g. Google Analytics) tracking code here -->

  <title>
    SARS-CoV-2 Genetic Interactions in Yeast - Methods
  </title>
  <meta name="viewport" content="width=device-width, initial-scale=1" />
  

<!-- https://metatags.io/ -->
<meta name="title" content="
SARS-CoV-2 Genetic Interactions in Yeast - Methods
" />
<meta name="description" content="A website on my ongoing research on SARS-CoV-2 genetic interactions in yeast in Rodney Rothstein's Lab at Columbia University." />
<meta property="og:type" content="website" />
<meta property="og:url" content="" />
<meta property="og:title" content="
SARS-CoV-2 Genetic Interactions in Yeast - Methods
" />
<meta property="og:description" content="A website on my ongoing research on SARS-CoV-2 genetic interactions in yeast in Rodney Rothstein's Lab at Columbia University." />
<meta property="twitter:card" content="summary_large_image" />
<meta property="twitter:url" content="" />
<meta property="twitter:title" content="
SARS-CoV-2 Genetic Interactions in Yeast - Methods
" />
<meta property="twitter:description" content="A website on my ongoing research on SARS-CoV-2 genetic interactions in yeast in Rodney Rothstein's Lab at Columbia University." />

<!-- RSS -->
<link
  rel="alternate"
  type="application/rss+xml"
  href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/feed.xml"
/>

<!-- https://developers.google.com/web/fundamentals/native-hardware/click-to-call#disable_auto-detection_when_necessary -->
<meta name="format-detection" content="telephone=no" />

  <!-- https://realfavicongenerator.net/ -->
<link
  rel="apple-touch-icon"
  sizes="180x180"
  href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/favicons/ios.png"
/>
<link
  rel="icon"
  type="image/png"
  sizes="512x512"
  href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/favicons/favicon.png"
/>
<link rel="manifest" href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/favicons/site.webmanifest" />

<!-- https://metatags.io/ -->
<meta
  property="og:image"
  content="/SARS-CoV-2-Genetic-Interactions-in-Yeast/favicons/share-thumbnail.jpg"
/>
<meta
  property="twitter:image"
  content="/SARS-CoV-2-Genetic-Interactions-in-Yeast/favicons/share-thumbnail.jpg"
/>

  <!-- Google Fonts -->
<link rel="preconnect" href="https://fonts.gstatic.com" />
<link
  href="https://fonts.googleapis.com/css2?family=Cousine:ital,wght@0,400;0,700;1,400;1,700&family=Lora:ital,wght@0,400;0,500;0,600;0,700;1,400;1,500;1,600;1,700&family=Poppins:ital,wght@0,400;0,500;0,600;0,700;1,400;1,500;1,600;1,700&display=swap"
  rel="stylesheet"
/>

<!-- Font Awesome (icons) -->
<link
  href="https://use.fontawesome.com/releases/v5.13.1/css/all.css"
  rel="stylesheet"
/>

  <!-- site-wide css styles -->
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/all.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/anchor.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/background.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/banner.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/card.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/citation.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/code.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/feature.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/figure.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/footer.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/gallery.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/genetic-interactions-list.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/header.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/heading.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/highlight.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/icon.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/image.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/input.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/link.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/list.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/paragraph.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/portrait.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/post-excerpt.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/post-info.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/post-nav.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/quote.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/role.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/rule.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/search-box.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/search-info.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/section.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/table.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/tags.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/tooltip.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/two-col.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/util.css" rel="stylesheet" />
  <!-- third party scripts -->
<script src="https://unpkg.com/@popperjs/core@2"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js"></script>

<!-- site-wide javascript scripts -->
<script src="/SARS-CoV-2-Genetic-Interactions-in-Yeast/js/anchors.js"></script>
  <script src="/SARS-CoV-2-Genetic-Interactions-in-Yeast/js/fetch-tags.js"></script>
  <script src="/SARS-CoV-2-Genetic-Interactions-in-Yeast/js/search.js"></script>
  <script src="/SARS-CoV-2-Genetic-Interactions-in-Yeast/js/tooltips.js"></script>
  <script src="/SARS-CoV-2-Genetic-Interactions-in-Yeast/js/true-hide.js"></script>
  

</head>

  <body>
    <header class="background" style="--background: url('/SARS-CoV-2-Genetic-Interactions-in-Yeast/images/background.jpg')" data-dark="true">
  <a
    href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/"
    class="logo_row"
    data-tooltip="Home"
  >
    <img src="/SARS-CoV-2-Genetic-Interactions-in-Yeast/images/tSNE.png" class="logo" />
    
      <span class="logo_text">SARS-CoV-2 Genetic Interactions in Yeast</span>
    
  </a>
  <nav class="nav_row"><a class="link" href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/background/" data-tooltip="Detailed background info"  data-text="true" data-style=""><span class="truncate">Background</span></a><a class="link" href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/methods/" data-tooltip="Approach, Methods, and Materials"  data-text="true" data-style=""><span class="truncate">Methods</span></a><a class="link" href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/preliminary%20findings/" data-tooltip="Current work"  data-text="true" data-style=""><span class="truncate">Preliminary Findings</span></a><a class="link" href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/genetic%20interactions/" data-tooltip="Code and Datasets"  data-text="true" data-style=""><span class="truncate">Screen Data</span></a><a class="link" href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/contributors/" data-tooltip="About and contact"  data-text="true" data-style=""><span class="truncate">Contributors</span></a></nav>
</header>

    <main>
      <section
      class="background"
      
      
    ><div class="section">

<h1 id="approach">Approach</h1>
<p>To screen for SDL interactions, the Rothstein group developed Selective Ploidy Ablation (SPA, Fig. 4) <sup> <a href="https://doi.org/10.1101/gr.109033.110">7</a> </sup>
. SPA involves a universal donor strain consisting of an over-expression plasmid and 16 modified chromosomes <sup> <a href="https://doi.org/10.1534/genetics.108.087999">8</a> </sup>
 destined for destabilization in diploid cells. After mating this donor strain with a library consisting of either single nonessential gene mutants or temperature sensitive alleles of essential genes, all of the chromosomes of the donor genome are selected against leaving only the over-expression plasmid and the recipient genome consisting of a single mutation (<a href="../#backgroundmotivation">Fig. 1A</a>). Mating and media transferring protocols are assisted via robotic pinning and growth of the recipient strains is measured through quantitating digital images of colony growth (<a href="../#backgroundmotivation">Fig. 1B</a>). The ScreenMill R package allows the quantification of colony sizes by utilizing image analyzing algorithms, colony size normalization, and manual input. After the review process, ScreenMill reports of screen results and relevant statistical information <sup> <a href="https://doi.org/10.1186/1471-2105-11-353">9</a> </sup>
. The coupling of SPA, ScreenMill, and the expression of SARS-CoV-2 proteins allows us to screen for SDL interactions. Strains of mutants that show an SDL interaction will be designated as hits and those will be explored further to determine the pathway and mechanism of action caused by viral proteins. The Rothstein laboratory is collaborating with virologist <a href="https://microbiology.columbia.edu/faculty-vincent-racaniello">Vincent Racaniello’s</a> group, which will verify SPA-discovered yeast pathways in mammalian cell lines using homologous genes.</p>

<figure class="figure">
  <a class="figure_image" style="">
    <img src="/SARS-CoV-2-Genetic-Interactions-in-Yeast/images/figure-4.png" alt="A. Mating of donor (black) strain with recipient (white) strain and subsequent donor chromosome destabilization. B. Plasmid transfer occurs through high-throughput robot pinning and 384 mutants per plate in quadruplicates overexpressing gene Y. C. Bottom magnification (over-expression of gene Y) shows an SDL for gene B as there is less growth compared to top magnification (no expression of gene Y)." title="A. Mating of donor (black) strain with recipient (white) strain and subsequent donor chromosome destabilization. B. Plasmid transfer occurs through high-throughput robot pinning and 384 mutants per plate in quadruplicates overexpressing gene Y. C. Bottom magnification (over-expression of gene Y) shows an SDL for gene B as there is less growth compared to top magnification (no expression of gene Y)." style="width: 400px;" loading="lazy" />
  </a><figcaption class="bold_figure_caption" align="justify">
      <b><i>Figure 4. SDL interaction screens using SPA.
</i></b>A. Mating of donor (black) strain with recipient (white) strain and subsequent donor chromosome destabilization. B. Plasmid transfer occurs through high-throughput robot pinning and 384 mutants per plate in quadruplicates overexpressing gene Y. C. Bottom magnification (over-expression of gene Y) shows an SDL for gene B as there is less growth compared to top magnification (no expression of gene Y).
</figcaption></figure>

</div></section><section
      class="background"
      
      
    ><div class="section">
<h1 id="methods-and-materials">Methods and Materials</h1>
<p><em>Note: This section is modified from a methods paper written by <a href="https://doi.org/10.1534/genetics.116.190231">Reid et. al</a>.</em>
<br /> <br /><strong>Yeast strains and methods</strong> <br />
Yeast gene disruption strain libraries in the BY4741 or BY4742 strain backgrounds for high throughput SDL screens were obtained from Thermo Fisher Scientific (Waltham, MA)<sup> <a href="https://doi.org/10.1126/science.285.5429.901">21</a> </sup>
. The yeast library of temperature-sensitive alleles for essential genes in the BY4741 background was a kind gift of Brenda Andrews (University of Toronto). Standard yeast growth media and culture techniques were used throughout this work<sup> <a href="https://searchworks.stanford.edu/view/9413810">22</a> </sup>
. Selective ploidy ablation (SPA) was used for high throughput transfer of expression plasmids into yeast strain libraries for SDL screens as previously described<sup> <a href="https://doi.org/10.1101/gr.109033.110">23</a> </sup>
. Colony growth was measured from scanned plate images using ScreenMill software as previously described<sup> <a href="https://doi.org/10.1186/1471-2105-11-353">24</a> </sup>
. Colony growth data processed via ScreenMill is provided <a href="../genetic%20interactions/#-genetic-interactions-determined-by-spa-screens">here</a>. Plates were manually reviewed to exclude regions of pinning failure, then colony measurements were normalized to the plate median. Replica pinning for the SPA procedure was accomplished using a Singer RoToR HDA robot (Singer Instruments, Somerset, UK). G418 and clonNAT were purchased from MediaTech and Werner Bioagents, respectively. LiOAc transformations were performed as described<sup> <a href="https://doi.org/10.1007/BF00340712">25</a> </sup>
.</p>

<p><br /><strong>Plasmid construction</strong> <br />
All plasmids were constructed using standard cloning techniques. The single and double gene expression empty vectors (pWJ1781, pWJ2216) was constructed as described in a previous paper by this group<sup> <a href="https://doi.org/10.1534/genetics.116.190231">26</a> </sup>
. To construct the galactose-inducible Nsp14 expression plasmid (pWJ2759) the Nsp14 open reading frame (ORF) was PCR amplified from a plasmid library of SARS-CoV-2 genes gifted by David Ho (Columbia University) using primers Nsp14-F1 (5′-GAATTCCAGCTGACCACCATGGCTGAAAATGTAACAGGACTTT-3′) and Nsp14-R1 (5′-GATCCCCGGGAATTGCCATGT- CATTACTGAAGTCTTGTAAAAGTGTTC-3′). Purified DNA was recombined in yeast with pWJ1781 plasmid DNA linearized by HpaI restriction digestion. Overexpression vectors containing Nsp14 exonuclease (D90A/E92A) and methyltransferase (D331A) mutants (pWJ2996, pWJ2997) were constructed via overlap PCR as described<sup> <a href="https://doi.org/10.1016/j.mex.2019.12.001">27</a> </sup>
 with primers found from <a href="#supplemental-material">Supplemental Table 1</a>. The dual expression vector of Nsp10 and Nsp14 (pWJ3008, pWJ3009, pWJ3010 for the ExoN and MT mutants) were constructed using the Nsp10 ORF cloned from the David Ho plasmid library with primers Nsp10-F (5′-GAATTCCAGCTGACCACCATGGCTGGTAATGCAACAGAAGTG-3′) and Nsp10-R (5′-GATCCCCGGGAATTGCCATGTCA- TTACTGAAGCATGGGTTCGCGG-3′). All plasmid constructs were verified by sequencing (GENEWIZ, South Plainfield, NJ).</p>

<p><br /><strong>Landscape enrichment analysis and statistics</strong> <br />
Landscape enrichment analysis was performed using genetic interaction profile correlations from The Cell Map<sup> <a href="https://doi.org/10.1534/g3.117.040220">28</a>, </sup>
<sup> <a href="https://doi.org/10.1126/science.aaf1420">29</a> </sup>
. The details regarding data exclusion, generation of t-SNE plots, and statistical calculations are as previous described<sup> <a href="https://doi.org/10.1093/nar/gkz631">30</a> </sup>
. MAT<i>α</i> and MAT<i>a</i> gene disruption SDL hits and suppressor hits were defined by a calculated Z-score ≤−2 and ≥1.5, respectively.</p>

<p><br /><strong>Colony growth and fitness measurements</strong> <br />
Overnight liquid cultures were grown to saturation in appropriate selective media. Saturated cultures were equilibrated to an OD600 of 1.0, arrayed in a 96-well microtiter plate, and pinned four times at 384 colony density giving 4 technical replicates using a Singer RoToR robot. Strains were grown on synthetic complete (SC) solid agar media with 2% galactose for induction and appropriate nutrient dropout to maintain plasmid selection. Plates containing the pinned colonies were incubated at 30C for 65h. For growth curves, colonies were scanned hourly. Colony density was measured using Screenmill<sup> <a href="https://doi.org/10.1186/1471-2105-11-353">24</a> </sup>
 and plotted using R with in-house scripts found <a href="../genetic%20interactions/#code">here</a>.</p>

</div></section><section
      class="background"
      
      
    ><div class="section">
<h1 id="supplemental-material">Supplemental Material</h1>
<p>Supplemental Table S1: <a href="http://sars-cov-2-genetic-interactions-in-yeast/methods/S1.csv">S1.csv</a></p>
</div></section>
    </main>
    <footer class="background" style="--background: url('/SARS-CoV-2-Genetic-Interactions-in-Yeast/images/background.jpg')" data-dark="true">
  <div class="footer_row"><a class="link" href="mailto:yinghan@caltech.edu" data-tooltip="Email" data-icon="true"  data-style=""><i class="fas fa-envelope fa-lg fa-fw"></i></a><a class="link" href="https://scholar.google.com/citations?hl=en&user=Megan Wang" data-tooltip="Google Scholar" data-icon="true"  data-style=""><i class="fab fa-google fa-lg fa-fw"></i></a><a class="link" href="https://github.com/meganwang08" data-tooltip="GitHub" data-icon="true"  data-style=""><i class="fab fa-github fa-lg fa-fw"></i></a></div>
  <!--
  <div class="footer_row">
    &copy; 2021 SARS-CoV-2 Genetic Interactions in Yeast
  </div>
  -->
</footer>

    <div class="tooltip">
  <div class="tooltip_content"></div>
  <div class="tooltip_arrow" data-popper-arrow></div>
</div>

  </body>
</html>
