<html>
  <head>
  <!-- put your analytics (e.g. Google Analytics) tracking code here -->

  <title>
    SARS-CoV-2 Genetic Interactions in Yeast - Preliminary Findings
  </title>
  <meta name="viewport" content="width=device-width, initial-scale=1" />
  

<!-- https://metatags.io/ -->
<meta name="title" content="
SARS-CoV-2 Genetic Interactions in Yeast - Preliminary Findings
" />
<meta name="description" content="A website on my ongoing research on SARS-CoV-2 genetic interactions in yeast in Rodney Rothstein's Lab at Columbia University." />
<meta property="og:type" content="website" />
<meta property="og:url" content="" />
<meta property="og:title" content="
SARS-CoV-2 Genetic Interactions in Yeast - Preliminary Findings
" />
<meta property="og:description" content="A website on my ongoing research on SARS-CoV-2 genetic interactions in yeast in Rodney Rothstein's Lab at Columbia University." />
<meta property="twitter:card" content="summary_large_image" />
<meta property="twitter:url" content="" />
<meta property="twitter:title" content="
SARS-CoV-2 Genetic Interactions in Yeast - Preliminary Findings
" />
<meta property="twitter:description" content="A website on my ongoing research on SARS-CoV-2 genetic interactions in yeast in Rodney Rothstein's Lab at Columbia University." />

<!-- RSS -->
<link
  rel="alternate"
  type="application/rss+xml"
  href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/feed.xml"
/>

<!-- https://developers.google.com/web/fundamentals/native-hardware/click-to-call#disable_auto-detection_when_necessary -->
<meta name="format-detection" content="telephone=no" />

  <!-- https://realfavicongenerator.net/ -->
<link
  rel="apple-touch-icon"
  sizes="180x180"
  href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/favicons/ios.png"
/>
<link
  rel="icon"
  type="image/png"
  sizes="512x512"
  href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/favicons/favicon.png"
/>
<link rel="manifest" href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/favicons/site.webmanifest" />

<!-- https://metatags.io/ -->
<meta
  property="og:image"
  content="/SARS-CoV-2-Genetic-Interactions-in-Yeast/favicons/share-thumbnail.jpg"
/>
<meta
  property="twitter:image"
  content="/SARS-CoV-2-Genetic-Interactions-in-Yeast/favicons/share-thumbnail.jpg"
/>

  <!-- Google Fonts -->
<link rel="preconnect" href="https://fonts.gstatic.com" />
<link
  href="https://fonts.googleapis.com/css2?family=Cousine:ital,wght@0,400;0,700;1,400;1,700&family=Lora:ital,wght@0,400;0,500;0,600;0,700;1,400;1,500;1,600;1,700&family=Poppins:ital,wght@0,400;0,500;0,600;0,700;1,400;1,500;1,600;1,700&display=swap"
  rel="stylesheet"
/>

<!-- Font Awesome (icons) -->
<link
  href="https://use.fontawesome.com/releases/v5.13.1/css/all.css"
  rel="stylesheet"
/>

  <!-- site-wide css styles -->
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/all.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/anchor.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/background.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/banner.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/card.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/citation.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/code.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/feature.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/figure.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/footer.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/gallery.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/genetic-interactions-list.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/header.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/heading.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/highlight.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/icon.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/image.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/input.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/link.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/list.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/paragraph.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/portrait.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/post-excerpt.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/post-info.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/post-nav.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/quote.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/role.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/rule.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/search-box.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/search-info.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/section.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/table.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/tags.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/tooltip.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/two-col.css" rel="stylesheet" />
      <link href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/css/util.css" rel="stylesheet" />
  <!-- third party scripts -->
<script src="https://unpkg.com/@popperjs/core@2"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js"></script>

<!-- site-wide javascript scripts -->
<script src="/SARS-CoV-2-Genetic-Interactions-in-Yeast/js/anchors.js"></script>
  <script src="/SARS-CoV-2-Genetic-Interactions-in-Yeast/js/fetch-tags.js"></script>
  <script src="/SARS-CoV-2-Genetic-Interactions-in-Yeast/js/search.js"></script>
  <script src="/SARS-CoV-2-Genetic-Interactions-in-Yeast/js/tooltips.js"></script>
  <script src="/SARS-CoV-2-Genetic-Interactions-in-Yeast/js/true-hide.js"></script>
  

</head>

  <body>
    <header class="background" style="--background: url('/SARS-CoV-2-Genetic-Interactions-in-Yeast/images/background.jpg')" data-dark="true">
  <a
    href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/"
    class="logo_row"
    data-tooltip="Home"
  >
    <img src="/SARS-CoV-2-Genetic-Interactions-in-Yeast/images/tSNE.png" class="logo" />
    
      <span class="logo_text">SARS-CoV-2 Genetic Interactions in Yeast</span>
    
  </a>
  <nav class="nav_row"><a class="link" href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/background/" data-tooltip="Detailed background info"  data-text="true" data-style=""><span class="truncate">Background</span></a><a class="link" href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/methods/" data-tooltip="Approach, Methods, and Materials"  data-text="true" data-style=""><span class="truncate">Methods</span></a><a class="link" href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/preliminary%20findings/" data-tooltip="Current work"  data-text="true" data-style=""><span class="truncate">Preliminary Findings</span></a><a class="link" href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/genetic%20interactions/" data-tooltip="Code and Datasets"  data-text="true" data-style=""><span class="truncate">Screen Data</span></a><a class="link" href="/SARS-CoV-2-Genetic-Interactions-in-Yeast/contributors/" data-tooltip="About and contact"  data-text="true" data-style=""><span class="truncate">Contributors</span></a></nav>
</header>

    <main>
      <section
      class="background"
      
      
    ><div class="section"><h1 id="preliminary-findings">Preliminary Findings</h1>
<h2 id="reduced-growth-resulting-from-nsp14-methyltransferase-domain">Reduced Growth Resulting From Nsp14 Methyltransferase Domain</h2>
<p>The effect of expressing nsp14 and both of its catalytic domains on strain growth was determined by comparison to controls containing the empty vector backbone (Fig. 5). To determine which Nsp14 domain is responsible for the slow growth seen in Fig. 5, separation of function mutants were constructed by inducing two well-studied point mutations through site-directed mutagenesis<sup> <a href="https://doi.org/10.1073/pnas.0808790106">17</a> </sup>
. The mutation of the ExoN domain conferred a growth defect statically identical to nsp14 WT expression, whereas this defect was absent while expressing the methyltransferase domain mutant.</p>

<figure class="figure">
  <a class="figure_image" style="width: 50%;">
    <img src="/SARS-CoV-2-Genetic-Interactions-in-Yeast/images/figure-5.png" alt="Growth curves are shown for wildtype strains grown at 30°C expressing either Nsp14, Nsp14 without a functional ExoN domain, Nsp14 without a functional MT domain, or an empty vector control." title="Growth curves are shown for wildtype strains grown at 30°C expressing either Nsp14, Nsp14 without a functional ExoN domain, Nsp14 without a functional MT domain, or an empty vector control." style="width: 100%;" loading="lazy" />
  </a><figcaption class="bold_figure_caption" align="justify">
      <b><i>Figure 5. Nsp14 methyltransferase domain is responsible for growth defect.
</i></b>Growth curves are shown for wildtype strains grown at 30°C expressing either Nsp14, Nsp14 without a functional ExoN domain, Nsp14 without a functional MT domain, or an empty vector control.
</figcaption></figure>

<p>Structural and functional analysis of the viral replisome shows that nsp10 binds to the nsp14 ExoN domain to stabilize and stimulate its activity (see <a href="background">background</a>). The nsp10-nsp14 complex assists in viral genome replication, so screening with expression of both proteins through a dual expression vector is necessary to better resemble what occurs during infection. Interestingly, co-expression of these two proteins attenuates the growth defect exhibited with expression of nsp14 alone (Fig. 6). This attenuation in growth allowed us to perform an SDL screen. We introduced nsp14 and its two mutants coupled with nsp10 into the nonessential mutant library. Again we find that the methyltransferase domain is responsible for the SDL interactions shown in Fig. 7. The two genetic clusters from the enriched regions in Fig. 7A and Fig. 7B correspond to mitochondrial and kinetochore-microtubule binding and histone proteins. Specifically, HHF1 and HHF2 encode for the subunits of H4, which is a core component of the nucleosome. These two hits suggest that the methyltransferase domain of nsp14 could be upregulating transcription of certain regions of the genome; however, further evidence of nsp14 methylation activity in yeast is needed to support this hypothesis.</p>

<figure class="figure">
  <a class="figure_image" style="width: 50%;">
    <img src="/SARS-CoV-2-Genetic-Interactions-in-Yeast/images/figure-6.png" alt="Growth curves are shown for wildtype strains grown at 30°C expressing either Nsp14, Nsp10 and Nsp14, or Nsp10 in combination with the two functional mutants. Expression of Nsp10 with Nsp14 attenuates growth defect in Nsp14 WT, ExoN and MT mutants, with the co-expressed MT mutant experiencing no significant growth defect." title="Growth curves are shown for wildtype strains grown at 30°C expressing either Nsp14, Nsp10 and Nsp14, or Nsp10 in combination with the two functional mutants. Expression of Nsp10 with Nsp14 attenuates growth defect in Nsp14 WT, ExoN and MT mutants, with the co-expressed MT mutant experiencing no significant growth defect." style="width: 100%;" loading="lazy" />
  </a><figcaption class="bold_figure_caption" align="justify">
      <b><i>Figure 6. Nsp10 and Nsp14 Co-expression attenuates growth defect.
</i></b>Growth curves are shown for wildtype strains grown at 30°C expressing either Nsp14, Nsp10 and Nsp14, or Nsp10 in combination with the two functional mutants. Expression of Nsp10 with Nsp14 attenuates growth defect in Nsp14 WT, ExoN and MT mutants, with the co-expressed MT mutant experiencing no significant growth defect.
</figcaption></figure>

<figure class="figure">
  <a class="figure_image" style="">
    <img src="/SARS-CoV-2-Genetic-Interactions-in-Yeast/images/figure-7.png" alt="**A-B.** MAT&lt;i&gt;a&lt;/i&gt; library screen consisting of ~4800 single nonessential gene mutations coexpressed with either nsp10-nsp14 wildtype, nsp10-nsp14 exonuclease mutation, or nsp10-nsp14 methyltransferase mutation. tSNE plots are organized through genetic interactions such that genes in the same pathway are closer to proximity with each other and redder heat map intensity indicates a higher number of hits per area. Existence of the methyltransferase domain leads to enrichment of two gene clusters (**A** and **B**), while the absence of a functional methyltransferase domain results in the loss of SDL interactions. " title="**A-B.** MAT&lt;i&gt;a&lt;/i&gt; library screen consisting of ~4800 single nonessential gene mutations coexpressed with either nsp10-nsp14 wildtype, nsp10-nsp14 exonuclease mutation, or nsp10-nsp14 methyltransferase mutation. tSNE plots are organized through genetic interactions such that genes in the same pathway are closer to proximity with each other and redder heat map intensity indicates a higher number of hits per area. Existence of the methyltransferase domain leads to enrichment of two gene clusters (**A** and **B**), while the absence of a functional methyltransferase domain results in the loss of SDL interactions. " style="" loading="lazy" />
  </a><figcaption class="bold_figure_caption" align="justify">
      <b><i>Figure 7. Methyltransferase domain leads to SDL hits seen in genetic landscape analysis of nsp10-nsp14 dual expression.
</i></b><strong>A-B.</strong> MAT<i>a</i> library screen consisting of ~4800 single nonessential gene mutations coexpressed with either nsp10-nsp14 wildtype, nsp10-nsp14 exonuclease mutation, or nsp10-nsp14 methyltransferase mutation. tSNE plots are organized through genetic interactions such that genes in the same pathway are closer to proximity with each other and redder heat map intensity indicates a higher number of hits per area. Existence of the methyltransferase domain leads to enrichment of two gene clusters (<strong>A</strong> and <strong>B</strong>), while the absence of a functional methyltransferase domain results in the loss of SDL interactions.
</figcaption></figure>

</div></section><section
      class="background"
      
      
    ><div class="section">
<h2 id="depletion-of-s-adenosyl-methionine-pools-rescues-nsp14-methyltransferase-domain-induced-growth-defect">Depletion of S-adenosyl Methionine Pools Rescues Nsp14 Methyltransferase Domain Induced Growth Defect</h2>
<p>Nsp14 is a SAM dependent methyltransferase, so depletion of SAM levels in the cell should affect Nsp14 methylation activity and present itself as attenuation of the Nsp14 growth defect in yeast. To alter SAM pools, a brief understanding of the SAM biosynethesis and recycling pathway is warranted (Fig 8, <a href="https://doi.org/10.1007/s00253-012-4261-3">18</a>). SAM synthesis and recycling in the cell starts with methionine metabolism as the formation of SAM is catalyzed by methionine adenosyltransferase (Sam1, Sam2). After SAM is utilized as a methyl donor, it is converted into S-adenosyl homocysteine (SAH). SAH is then processed into adenosine and homocysteine (Hcy) facilitated by reversible SAH hydrolase (Sah1). The adenosine and Hcy are recycled into AMP and methionine by adenosine kinase (Ado1) and methionine synthase (Met6) respectively.</p>

<p>Ado1 deletions strains are shown to have higher intercellular concentrations of adenosine. Excess adenosine reverses the reaction driven by Sah1 - leading to an accumulation of SAH and a decrease in SAM levels as the substrates in the recycling pathway exist as SAH <sup> <a href="https://doi.org/10.1016/j.bbadis.2012.09.007">19</a> </sup>
. Coupled with the effect of reducing SAM concentrations, SAH also competitively inhibits methyltransferases. Due to its structural similarities to SAM, SAH can bind reversibly with methyltransferases to temporarily block the catalytic site and slow activity <sup> <a href="https://doi.org/10.1016/j.tips.2004.04.004">20</a> </sup>
.</p>

<p>The growth defect induced by Nsp14 methyltransferase activity was tested by leveraging increased levels of SAH resulting from ADO1 deletion (Fig. 8). Overexpression of Nsp14 in the ΔADO1 strain leads to similar growth as the empty vector control while ΔHIS3 ADO1+ control strains with WT Nsp14 cause a significant growth defect. Nsp14 MT mutants also exhibit no growth defect in the ΔADO1 strains but show a slight growth defect in the ΔHIS3 ADO1+ strain. These results provide more evidence that Nsp14 is exhibiting methyltransferase activity in our model system.</p>

<figure class="figure">
  <a class="figure_image" style="">
    <img src="/SARS-CoV-2-Genetic-Interactions-in-Yeast/images/figure-8.png" alt="**A.** Schematic illustration of the metabolic pathways involved in methionine biosynthesis in S. cerevisiae. **B.** Growth curves are shown for wildtype strains grown at 30°C expressing Nsp14 in ΔADO1 or ΔHIS3 ADO1+ strains. Increased levels of SAH reduce the growth defect caused by Nsp14 methylation activity. The exonuclease mutant (data not shown) presented with no significant difference in growth between the two strains." title="**A.** Schematic illustration of the metabolic pathways involved in methionine biosynthesis in S. cerevisiae. **B.** Growth curves are shown for wildtype strains grown at 30°C expressing Nsp14 in ΔADO1 or ΔHIS3 ADO1+ strains. Increased levels of SAH reduce the growth defect caused by Nsp14 methylation activity. The exonuclease mutant (data not shown) presented with no significant difference in growth between the two strains." style="" loading="lazy" />
  </a><figcaption class="bold_figure_caption" align="justify">
      <b><i>Figure 8. Overexpression of Nsp14 in Ado1 deletion strains suppresses Nsp14 induced growth defect.
</i></b><strong>A.</strong> Schematic illustration of the metabolic pathways involved in methionine biosynthesis in S. cerevisiae. <strong>B.</strong> Growth curves are shown for wildtype strains grown at 30°C expressing Nsp14 in ΔADO1 or ΔHIS3 ADO1+ strains. Increased levels of SAH reduce the growth defect caused by Nsp14 methylation activity. The exonuclease mutant (data not shown) presented with no significant difference in growth between the two strains.
</figcaption></figure>

</div></section><section
      class="background"
      
      
    ><div class="section">
<h2 id="encountered-difficulties-in-the-project">Encountered Difficulties in the Project</h2>
<p>To directly confirm methylation activity in our system, I need to conduct an assay that compares methylation between yeast strains expressing Nsp14 and a control vector. However, radioactive SAM is expensive and I wanted to conduct more confirmatory experiments before going down that route. I am currently conducting growth curves on series of plates titrated with SAH and SAM. I expect that alterations to the endogenous SAM pools will result in various activities of the Nsp14 methyltransferase domain leading to differences in overall growth in our system.</p>

<p>Additionally, in my exploration of the mitochondrial hits from the Nsp10-Nsp14 SPA screen, I have been expressing Nsp14 under a copper promoter to validate those hits. I am first conducting growth curves to see what induction of copper leads to adequate expression of Nsp14 for our screen and then will run a western blot at those concentrations. However, I have yet to find a concentration of copper that consistently leads to Nsp14 induced growth defects (in biological replicates). In my next growth curve, I will set up positive control using a copper-driven plasmid containing an endogenous yeast protein that, when overexpressed, causes a growth defect.</p>
</div></section>
    </main>
    <footer class="background" style="--background: url('/SARS-CoV-2-Genetic-Interactions-in-Yeast/images/background.jpg')" data-dark="true">
  <div class="footer_row"><a class="link" href="mailto:yinghan@caltech.edu" data-tooltip="Email" data-icon="true"  data-style=""><i class="fas fa-envelope fa-lg fa-fw"></i></a><a class="link" href="https://scholar.google.com/citations?hl=en&user=Megan Wang" data-tooltip="Google Scholar" data-icon="true"  data-style=""><i class="fab fa-google fa-lg fa-fw"></i></a><a class="link" href="https://github.com/meganwang08" data-tooltip="GitHub" data-icon="true"  data-style=""><i class="fab fa-github fa-lg fa-fw"></i></a></div>
  <!--
  <div class="footer_row">
    &copy; 2021 SARS-CoV-2 Genetic Interactions in Yeast
  </div>
  -->
</footer>

    <div class="tooltip">
  <div class="tooltip_content"></div>
  <div class="tooltip_arrow" data-popper-arrow></div>
</div>

  </body>
</html>
